src/app.ts

```diff
@@ -11,7 +11,7 @@ const env = process.env["ENV"] ?? "";
 const app = express();
 app.use(
   cors({
-    origin: `https://${env === "production" ? "" : `${env}.`}permanent.org`,
+    origin: `https://${env === "production" ? "www" : `${env}.`}permanent.org`,
```

Comment:
I think we need a `.` prefixing the `permanent.org` now (in place of the `.` after `${env}`

Reply:
oh thanks!